images/sqlx-cli/Dockerfile

```diff
@@ -1,22 +1,21 @@
 ################################################################################
 # Builder
 ################################################################################
-FROM rust:1.78-alpine AS builder
+FROM rust:1.85-alpine AS builder
```

Comment:
We will forget to increment this version and risk hitting rustc MSRV issues every time someone updates `sqlx` dependency. I left a comment on original (reverted) PR that suggests how to template this version number to keep it in sync with our `rust-toolchain` file.

Reply:
Just for record: https://github.com/kamu-data/kamu-cli/pull/1205#issuecomment-2817434338

Reply:
Done